LOADING_SUPPORT.md

```diff
@@ -0,0 +1,185 @@
+# Setting up bulk loading support for a database type
```

Comment:
Since I was walking through the process, this seemed like a good point to document how to do this!